frontend/integration-tests/protractor.conf.ts

```diff
@@ -230,7 +230,8 @@ export const checkErrors = async () =>
     }
   });
 
-export const firstElementByTestID = (id: string) => $(`[data-test-id=${id}]`);
+export const firstElementByTestID = (id: string) =>
+  element.all(by.css(`[data-test-id=${id}]`)).first();
```

Comment:
Is this to avoid the Protractor warning only?

Reply:
yes, and I just got rid of it in lieu of specifically setting 'dropdown-menu-<receiverform>' which was throwing the Protractor warning.

Reply:
fixed